python/dask_cudf/dask_cudf/batcher_sortnet.py

```diff
@@ -3,11 +3,25 @@
 Adapted from https://en.wikipedia.org/wiki/Batcher_odd%E2%80%93even_mergesort
 """
 import math
+from operator import getitem
+
+import numpy as np
 
 from dask import compute, delayed
+from dask.base import tokenize
+from dask.dataframe.core import DataFrame, _concat
+from dask.dataframe.shuffle import rearrange_by_column_tasks
+from dask.dataframe.utils import group_split_dispatch, hash_object_dispatch
+from dask.highlevelgraph import HighLevelGraph
+from dask.utils import digit, insert
 
 import cudf as gd
 
+try:
+    import cytoolz as toolz
+except ImportError:
+    import toolz
```

Comment:
As I just learned yesterday, apparently we can just do...

```suggestion
    import tlz as toolz
```

<br>

xref: https://github.com/dask/dask/pull/5949#issuecomment-595407707

Reply:
sweet.